win32ss/reactx/dxgthk/dxgthk.spec

```diff
@@ -1,3 +1,4 @@
+@ stdcall DriverEntry(ptr ptr)
```

Comment:
Isn't the entry-point usually automatically "exported" ?

Reply:
No, it isn't. If to open our dxgthk.sys on recent ROS master via Dependency Walker, it isn't exported at all, unlike Windows:
![ros_no_export](https://github.com/reactos/reactos/assets/26385117/ce101017-d292-4659-b88d-f2d2bc1d6d73)